pipeline_noted.ipynb

```diff
@@ -480,18 +576,37 @@
    "source": [
     "**In production mode** -- pipeline.ipynb or batch_processing.ipynb, we usually use the `subset` parameter defined above to control this. It a good chance for us to recall the `subset` parameter. \n",
     "```python\n",
+    "subset = None\n",
+    "```\n",
+    " `subset` is used to subset the data, which operates on the whole data. `subset` should be specified as python [dictionary](https://docs.python.org/3/tutorial/datastructures.html#dictionaries), whose `keys` are the dimension along which subsetting should be done, and `values` specify how subsetting should be done (usually a [slice object](https://docs.python.org/3/c-api/slice.html)). A good usecase for this is to take out some troublesome frames or take out some bad pixels, such as those used as timestamps in wireless miniscope recording.\n",
+    "<div class=\"alert alert-success\">                                                               \n",
+    "Here is good chance to introduce how to create a dictionary. The basic format of a dictionary will be like\n",
+    "</div>\n",
+    "\n",
+    "```python\n",
+    "dictionary = {'key1': value1, 'key2': value2, ... 'keyN': valueN}\n",
+    "```\n",
+    "\n",
+    "<div class=\"alert alert-success\">                                                               \n",
+    "For example, if you only want to keep the first 800 frames, and the height and width from 100 to 200, you can create the subset dictionary like this:\n",
```

Comment:
want to introduce slice object as well? (and say it doesn't has to be a slice object -- anything `.sel` can take would be fine?)